test/OpenTelemetry.Exporter.Prometheus.Tests/PrometheusSerializerTests.cs

```diff
@@ -96,6 +96,34 @@ public void GaugeZeroDimensionWithUnit()
             Assert.Matches(
                 ("^"
                     + "# TYPE test_gauge_seconds gauge\n"
+                    + "# UNIT test_gauge_seconds seconds\n"
+                    + "test_gauge_seconds 123 \\d+\n"
+                    + "$").Replace('\'', '"'),
+                Encoding.UTF8.GetString(buffer, 0, cursor));
+        }
+
+        [Fact]
+        public void GaugeZeroDimensionWithDescriptionAndUnit()
+        {
+            var buffer = new byte[85000];
+            var metrics = new List<Metric>();
+
+            using var meter = new Meter(Utils.GetCurrentMethodName());
+            using var provider = Sdk.CreateMeterProviderBuilder()
+                .AddMeter(meter.Name)
+                .AddInMemoryExporter(metrics)
+                .Build();
+
+            meter.CreateObservableGauge("test_gauge", () => 123, unit: "seconds", description: "Hello, world!");
```

Comment:
nice description :)

Reply:
Needs some clever Greek mythology thing. Defer to @alanwest to come up with it 🤣

Reply:
I mean, would "Hello, Prometheus!" not have been clever enough? And also what the heck do I know about Greek mythology?

Reply:
Do we consider the Prometheus Exporters to be renamed to OpenMetrics Exporters? 😆